proposals/stage-0/individual-membership-director/README.md

```diff
@@ -0,0 +1,73 @@
+## Individual Membership Representation Proposal
+
+Stage 0
+
+### Champion
+[Tracy Hinds]()
+
+### Description
+
+Board representation
+
+The Individual Membership program will bring together the ecosystem around $FOUNDATION projects, users who are not current contributors, that we want to feel more engaged and welcoming than other languages, through participation, input, and accessing the benefits of the foundation that have historically been perceived as available only to contributors. This program codifies a relationship with our ecosystem, as an opportunity for member companies to feel more connected with the members of the Individual Membership through systematic communication building.
+
+In order for the Individual Membership Director to provide high fidelity context on the Board, this seat on the Board would only exist with a threshold contingency: once the individual membership roster hits 2000 active members, an election would be triggered for representation on the $FOUNDATION Board of Directors. The term of the elected seat would be for one year. When the year terminates, only if the 2000 person threshold is maintained would a new election occur.
+
+The process for updating the Individual Membership Director seat requires addressing:
+
+- Who is eligible to nominate and run for a Directorship role 
+- Who is eligible to vote for candidates for a Directorship role 
+  
+
+The Individual Membership Director candidates are to be nominated by voting members of the top-level committees of projects and the CPC. The active members of the Individual Membership program would then vote via our proposed election system on the candidate of their choosing.
+
```

Comment:
Would nominees be limited to people who had paid individual membership dues, and if so, what class of membership do they have to be (any/bronze/silver/etc)?

Reply:
Excellent question. In the past, you did have to be an individual member to be nominated. I think that makes sense and there's a very low hurdle with this new model. I don't think it should be restricted by tier, so I'd say _any_. If that sounds good, I'll clarify in this line.